src/services/auth/utils.ts

```diff
@@ -1,10 +1,17 @@
-import { CLIENT_HOST, CLIENT_HOSTS } from '../../utils/config';
+import { Context } from '@graasp/sdk';
+
+import { CLIENT_HOSTS } from '../../utils/config';
+
+const defaultClientHost = CLIENT_HOSTS.find((c) => c.name === Context.Builder);
+if (!defaultClientHost) {
+  throw new Error('Default Builder client host environment variable not set!');
+}
 
 export const getRedirectionUrl = (url?: string) => {
   // TODO: improve check
   // TODO: whitelist graasp domains for redirection
   if (!url || !CLIENT_HOSTS.some(({ hostname }) => url.startsWith(hostname))) {
-    return CLIENT_HOST;
+    return defaultClientHost.hostname;
   }
   return url;
 };
```

Comment:
Okay I found the issue. The tests had some default behaviour when `CLIENT_HOST` was undefined (as it's the case in the CI) so now that I've removed `CLIENT_HOST` and always provide `BUILDER_CLIENT_HOST` instead, the auth tests should always receive redirection on success. On failure, it depends on `AUTH_CLIENT_HOST`